src/main.rs

```diff
@@ -50,7 +50,37 @@ ow=0:or=0;38;5;16;48;5;203:no=0:ex=1;38;5;203:cd=0;38;5;203;48;5;236:mi=0;38;5;1
 ";
 
 fn run() -> Result<ExitCode> {
-    let matches = app::build_app().get_matches_from(env::args_os());
+    let mut app = app::build_app();
+
+    // When encountering parsing errors, print error message and exit
+    let matches = match app.get_matches_from_safe_borrow(env::args_os()) {
```

Comment:
Could you explain why that change is necessary? get_matches_from => get_matches_from_safe_borrow

Reply:
Calling `get_matches_from` consumes `app` and thus `app` can not be used to call `print_help`:

![](https://i.imgur.com/brGPOuA.png)

Reply:
Ok, thanks.